src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1366,7 +1366,7 @@ public static bool IsPartial(this TypeSymbol type)
 
         public static bool HasFileLocalTypes(this TypeSymbol type)
         {
-            var foundType = type.VisitType(predicate: (type, _, _) => type is SourceMemberContainerTypeSymbol { IsFileLocal: true }, arg: (object?)null);
+            var foundType = type.VisitType(predicate: (type, _, _) => type.OriginalDefinition is SourceMemberContainerTypeSymbol { IsFileLocal: true }, arg: (object?)null);
```

Comment:
Could we just do `=> type.IsFileLocal` now?

Reply:
Oops, the whole point was to start using the new API here.